backend/fregepoc/celery_app.py

```diff
@@ -5,7 +5,7 @@
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "fregepoc.settings")
 
-app = Celery("frege")
+app = Celery("frege", worker_max_tasks_per_child=4)
```

Comment:
Wouldn't it be more appropriate to read this parameter from env vars? What do you think?